### PR TITLE
Session Patch Description

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Sessions/SessionsController.fs [hunk 1]
```diff
@@ -7,13 +7,20 @@ open System.Web.Http
...
+
     member x.Post(session: Session) = Catch.respond x HttpStatusCode.Created (fun () -> session |> Session.toEntity |> add)
 
     member x.Get() = Catch.respond x HttpStatusCode.OK (fun () -> getAll() |> Seq.map Session.toModel)
 
-    member x.Get(id : Guid) = Catch.respond x HttpStatusCode.OK (fun () -> get id |> Session.toModel)
\ No newline at end of file
+    member x.Get(id : Guid) = Catch.respond x HttpStatusCode.OK (fun () -> get id |> Session.toModel)
+
+    member x.Patch(id: Guid, op: PatchOp) = Catch.respond x HttpStatusCode.NoContent (fun () -> patch id op)
```
Comment (pallenSL) [inline Sessions/SessionsController.fs]: What does this do if I pass ?id=xxx&op=&quot;bye&quot;TRUNCATE%20TABLE%20sessions%20WITH%20NO%20LOG;GO;

Just askin'.  :D

Comment (jebbinSCL) [inline Sessions/SessionsController.fs]: That specifically wouldn't parse! it would come in as null and throw an error.

Comment (pallenSL) [inline Sessions/SessionsController.fs]: What about Patching a session that doesn't exist?
Are REST return codes going to be a separate task?

Comment (jebbinSCL) [inline Sessions/SessionsController.fs]: I'll adjust the code for one's that don't exist. 
Yup, rest return codes will be separate.

Comment (jebbinSCL) [inline Sessions/SessionsController.fs]: I've adjusted the code for a session that doesn't exist.

**Comment by pallenSL** [comment]
:shipit:
:boat:

**Pull request merged**